lib/msal-core/samples/quickStart/Readme.txt

```diff
@@ -0,0 +1,15 @@
+
+How to run the dev apps:
+--------------------
+Pre-requisite
+- Install node.js if needed (https://nodejs.org/en/)
+
+Resolving the server.js references
+- In a command prompt, run npm install
+
+Running the sample
+- In a command prompt, run �node server.js�
```

Comment:
There seem to be certain special chars in the markdown. Can we replace it with back ticks?